tests/Feature/MovieDeleteTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Movie;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class MovieDeleteTest extends TestCase
+{
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function test_movie_delete_authurised_user_can_remove_movie()
```

Comment:
Another test: 
- guest cannot delete movie
- movies that do not exist are not deleted.

Any tests that involve changing something in the database, we want to check if the database is the way we expected. :D